boards/arm/atsame54_xpro/doc/index.rst

```diff
@@ -105,6 +107,13 @@ this BSP. SERCOM2 is the default Zephyr console.
 
 - SERCOM2 115200 8n1 connected to the onboard Atmel Embedded Debugger (EDBG)
 
+PWM
+===
+
+The SAME54 MCU has 5 TCC based PWM units with up to 4 outputs each and a period
```

Comment:
6 outputs?

Reply:
Yup, forgot to update the docs. Will fix.

Reply:
Fixed